.github/workflows/release-linux.yml

```diff
@@ -119,12 +116,24 @@ jobs:
         uses: docker/setup-buildx-action@v3
 
       - name: Login to Docker Hub
+        if: "${{ vars.DOCKERHUB_USERNAME != '' }}"
         uses: docker/login-action@v3
         with:
           username: ${{ vars.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Build
```

Comment:
Its only split up since i cant think of an easy way to keep them combined that accomplishes the end goal. I'll play around with this and see if we can keep this as a single piece.

Reply:
Converting to draft until i can figure out a good way to merge them.